<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALCInterface</class>
 <widget class="QMainWindow" name="ALCInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALC Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stepView">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="dataLoadingView"/>
     <widget class="QWidget" name="baselineModellingView"/>
     <widget class="QWidget" name="peakFittingView"/>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1,0">
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
        <item>
          <widget class="QPushButton" name="exportResults">
            <property name="text">
              <string>Export results...</string>
            </property>
          </widget>
        </item>
        <item>
        <widget class="QPushButton" name="nextStep">
         <property name="text">
          <string>%NEXT%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>%LABEL%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="previousStep">
         <property name="text">
          <string>%PREV%</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
